subprojects/docs/src/docs/release/notes.md

```diff
@@ -164,6 +164,22 @@ On case-insensitive file systems (e.g. NTFS and APFS), a file/folder rename wher
 For example, renaming an input of a `Copy` task called `file.txt` to `FILE.txt` will now cause `FILE.txt` being created in the destination directory. 
 The `Sync` task and `Project.copy()` and `sync()` operations now also handle case-renames as expected.
 
+## Unavailable files are handled more gracefully
+Generally, broken symlinks, named pipes and unreadable files/directories (hereinafter referred to as unavailable files)
+found in inputs/outputs of tasks are handled gracefully from now on: similarly to missing files.
+
+In incremental builds therefore, changing the availability of a file (e.g. modifying the target of a broken symlink to a valid file, or vice versa),
```

Comment:
I think all this documentation should go to `more_about_tasks.adoc` in a new section talking about symbolic links under the `sec:up_to_date_checks`. The documentation about caching could go in the build cache guide under [symbolic links](https://guides.gradle.org/using-build-cache/#symbolic_links) or also in the up-to-date checks section.

WDYT?

Reply:
> The documentation about caching could go in the build cache guide

I understood that we should rather remove documenting the cache and the skipwhenempty parts, since it's confusing and will change in 6 anyway.

Reply:
Done, please check if it's what you've requested